crates/extensions/c8y_mapper_ext/src/tests.rs

```diff
@@ -1176,7 +1176,7 @@ async fn inventory_registers_unknown_entity_once() {
 
     let measurement_message = MqttMessage::new(
         &Topic::new("te/device/main/service/my_service/m/measurement").unwrap(),
-        "",
+        r#"{"foo":25}"#,
```

Comment:
A proper measurements is required otherwise the registration message is rejected by the converter.